src/mdx/FormViewer.js

```diff
@@ -0,0 +1,38 @@
+import { Form } from "@bpmn-io/form-js";
+//import '@bpmn-io/form-js-carbon-styles/src/carbon-styles.scss';
+import React from "react";
+
+import "../css/formViewer.css";
+
+export default class FormViewer extends React.Component {
+  formContainer;
+
+  componentDidMount() {
+    const schema = this.props.schema || {};
+    const data = this.props.data || {};
+
+    const form = new Form({
+      container: this.formContainer,
+    });
+
+    form.importSchema(schema, data);
+  }
+
+  render() {
+    const customClass = this.props.customClass || "";
+
+    return React.createElement("div", {
```

Comment:
Why are you using `createElement` and a class component? MDX doesn't support JSX and function components?

Reply:
MDX does, but the component is plain JS; I tried to create a JSX file but couldn't manage to make it work. There should be a way to do so in Docusaurus, but I am not sure how

I'd keep that as a nice-to-have improvement for now, ok? @vsgoulart